internal/testutils/networking/host_addr_linux.go

```diff
@@ -0,0 +1,52 @@
+package networking
+
+import (
+	"net"
+	"syscall"
+	"testing"
+
+	"github.com/docker/docker/internal/nlwrap"
+	"github.com/vishvananda/netlink"
+	"gotest.tools/v3/assert"
+)
+
+// FindHostAddr returns the address of the host machine that would be used to
+// reach external hosts. family can be either unix.AF_INET, or unix.AF_INET6.
+func FindHostAddr(t *testing.T, family int) string {
+	t.Helper()
+
+	var dst string
+	// These are IPv4 and IPv6 addresses of Cloudflare's DNS servers. These
+	// don't really matter - we just need to find a route to some external
+	// server.
+	if family == netlink.FAMILY_V4 {
+		dst = "1.1.1.1"
+	} else if family == netlink.FAMILY_V6 {
+		dst = "2606:4700:4700::1111"
+	} else {
+		t.Fatalf("unknown address family %d", family)
+	}
```

Comment:
Must it be able to connect to these? Considering;

- https://github.com/moby/moby/issues/47541

Reply:
🙈 never mind this is for testing only ignore me ignore me

Reply:
Yeah, and we don't even resolve these IPs, or connect to them - so nothing leaves the netns. It's just that I needed some publicly routed IP addresses to do a routing table lookup.